omod/src/main/resources/config.xml

```diff
@@ -228,6 +232,12 @@
 		<description>Set observation strategy.</description>
 	</globalProperty>
 
+	<globalProperty>
+		<property>${project.parent.artifactId}.condition.strategy</property>
+		<defaultValue>DefaultConditionStrategy</defaultValue>
```

Comment:
Is this just a key or a call with full package name?

Reply:
It's a call with full package name

Reply:
Did you intentionally leave out the packages from the default value?

Reply:
I Don't understand what you mean by that.